packages/side-nav/src/SideNav.spec.tsx

```diff
@@ -283,6 +285,17 @@ describe('packages/side-nav', () => {
         expect(childContent).toBeInTheDocument();
       });
     });
+
+    describe('when `collapsible` is true and `defaultCollapsed` is false', () => {
```

Comment:
would we want to make `collapsible={false} defaultCollapsed={true}` be a type error?

Reply:
Also good call